src/main/java/venus/Storage.java

```diff
@@ -0,0 +1,136 @@
+package venus;
+
+/**
+ * This is a Storage class that controls data to be stored or loaded from .txt data file.
+ *
+ * @author peterXGD
+ * @since 2024-02-05
+ */
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
```

Comment:
Instead of working with file paths as strings, you can consider using Path for better abstraction and manipulation.
```suggestion
    Path filePath = Paths.get("data", "venus.txt");
```